src/Containers/WeatherToday/weatherToday.tsx

```diff
@@ -1,130 +1,72 @@
 import React, {useEffect, useState} from "react";
-import axios from "axios";
 import {WeatherCard} from "./components/WeatherCard/WeatherCard";
 import {SearchBar} from "./components/searchBar/searchBar";
 import {PreloaderComponent} from "../../Component/preloader/preloader";
+import {getWeatherToday} from "../../api/weatherAPI";
+import {autoLocationWeatherDisplay} from "../../api/weatherAPI";
+
+interface weatherStateInterface {
+    cityTitle: string,
+    humidity: string,
+    feelsLike: number,
+    temperature: number,
+    pressure: number,
+    timeSunrise: string,
+    timeSunset: string,
+}
 
 export const WeatherToday = (): JSX.Element => {
-    const [cityTitleValue, setCityTitleValue] = useState<string>('')
-    const [cityTitle, setCityTitle] = useState<string>('')
-    const [sunrise, setSunrise] = useState<string>('')
-    const [sunset, setSunset] = useState<string>('')
-    const [weatherState, setWeatherState] = useState<object>({
-        cityTitle: '',
-        humidity: '',
-        feelsLike: 0,
-        temperature: 0,
-        pressure: 0,
-})
+    const [cityTitleValue, setCityTitleValue] = useState<string>("")
     const [isLoading, setIsLoading] = useState<boolean>(false)
-    const apiKEY = '362efd6a69303a0f5662a19ce25e5e89'
-    let filterTimeout: NodeJS.Timeout;
+    const [weather, setWeather] = useState<weatherStateInterface | undefined>()
+    let filterTimeout: NodeJS.Timeout | undefined;
 
     useEffect(() => {
-        setIsLoading(true)
-        getCurrentPosition()
+        locatePosition()
     }, [])
 
-    const doCityFilter = (query: string) => {
+    const CityFilter = (filterCity: string) => {
```

Comment:
functions should not have capitalized names, just filterCity = (filter: string) => {...}